corehq/apps/es/__init__.py

```diff
@@ -4,6 +4,7 @@
 from . import domains
 from . import forms
 from . import users
+from . import queries
```

Comment:
That's adding it to the module-level interface, so you can have it available from
`from corehq.apps import es`